connexion/json_schema.py

```diff
@@ -128,6 +128,29 @@ def nullable_validation_fn(validator, to_validate, instance, schema):
     return nullable_validation_fn
 
 
+def validate_required(validator, required, instance, schema):
+    if not validator.is_type(instance, "object"):
+        return
+
+    for prop in required:
+        if prop not in instance:
+            properties = schema.get('properties')
+            if properties is not None:
+                subschema = properties.get(prop)
+                if subschema is not None:
+                    if 'readOnly' in validator.VALIDATORS and subschema.get('readOnly'):
+                        continue
+                    if 'writeOnly' in validator.VALIDATORS and subschema.get('writeOnly'):
+                        continue
+                    if 'x-writeOnly' in validator.VALIDATORS and subschema.get('x-writeOnly') is True:
+                        continue
+            yield ValidationError("%r is a required property" % prop)
+
+
+def validate_readOnly(validator, wo, instance, schema):
+    yield ValidationError("Property is read-only")
```

Comment:
@sampoh0523 has convinced me that there is no need for this function, the argument is that Connexion should tolerate a readOnly field in requests without complaint.